### PR TITLE
 Bundle with Rollup

### PR DESCRIPTION
**Pull request by keller-mark**
I updated the call to `onReady` in `demo/src/renderDemo.js` https://github.com/hms-dbmi/vitessce-grid/compare/master...keller-mark:keller-mark/rollup?expand=1#diff-5e483e61998cb027af5358489f694e19R22 since I got this error:

Screen Shot 2020-05-10 at 9 56 22 PM

I updated eslint dependencies and versions to reflect those in `vitessce`.

I was not able to figure out how to disable caching of files during Karma autoWatch tests but added comments about what I tried here: https://github.com/hms-dbmi/vitessce-grid/compare/master...keller-mark:keller-mark/rollup?expand=1#diff-0b85293f3f3c71d61eb133efcd7ebb3fR37

Not sure whether the broken autoWatch tests are blockers.

It seems that there is no output `css` directory, since there is only CSS is in the demo directory https://github.com/hms-dbmi/vitessce-grid/compare/master...keller-mark:keller-mark/rollup?expand=1#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L8

Switched to using a function to generate the demo HTML file https://github.com/hms-dbmi/vitessce-grid/compare/master...keller-mark:keller-mark/rollup?expand=1#diff-3ca5b53d595296be61a149e60743962cR27 since that is what the rollup `html` plugin supports.

### CODE REVIEW

**Comment by keller-mark** [comment]
cc @ilan-gold @mccalluc

File: rollup.utils.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    ${links}
+  </head>
+  <body>
+    <div id="demo"></div>
+    <noscript>
+    You need to enable JavaScript to run this app.
+    </noscript>
+    ${scripts}
+  </body>
+</html>`;
```
Comment (ilan-gold) [inline rollup.utils.js:33]: This is probably ok for this, but we would want something more maintainable for vitessce....

File: rollup.config.lib.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        // Reference: https://github.com/rollup/plugins/tree/master/packages/node-resolve#using-with-rollupplugin-commonjs
+        commonjs({
+            // Using this RegEx rather than 'node_modules/**' is suggested, to enable symlinks.
+            // Reference: https://github.com/rollup/plugins/tree/master/packages/commonjs#usage-with-symlinks
+            include: /node_modules/,
+            namedExports: {
+                // Need to explicitly tell Rollup how to handle imports like `React, { useState }`
+                // Reference: https://github.com/rollup/rollup-plugin-commonjs/issues/407#issuecomment-527837831
+                // Reference: https://github.com/facebook/react/issues/11503
+                'node_modules/react/index.js': Object.keys(React)
```
Comment (ilan-gold) [inline rollup.config.lib.js:87]: Why not also react-dom?

**Comment by ilan-gold** [comment]
I wonder if that error is at the root of what was going on with what I was seeing....

File: rollup.config.lib.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+const resolve = require('@rollup/plugin-node-resolve');
+const { babel } = require('@rollup/plugin-babel');
+const json = require('@rollup/plugin-json');
+const commonjs = require('@rollup/plugin-commonjs');
+const { terser } = require('rollup-plugin-terser');
+
+// Constants for output files.
+const SRC_DIR = 'src';
+const LIB_DIR = 'lib';
```
Comment (ilan-gold) [inline rollup.config.lib.js:13]: Maybe a more descriptive name for cjs unless this is convention.

File: rollup.config.lib.js (hunk 2 above)
Comment (ilan-gold) [reply rollup.config.lib.js:87]: Ah, I see the above comment now.

File: rollup.utils.js (hunk 1 above)
Comment (keller-mark) [reply rollup.utils.js:33]: True

Comment (keller-mark) [reply rollup.utils.js:33]: There must be a better html plugin for rollup that can handle templates as `.html` files.

**Pull request merged**